website/mails/mail_callbacks.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```

Comment:
When you rename `callback` to `presend`, don't forget to rename this file, too. I think `presend.py` makes sense.

Alternatively, it might be more cohesive to put all queue mail code in its own module (separate from the "normal" emails). For example, the QueuedMail model, the queued email "structs", and these presend functions could go in a module `website.mails.queued`.